formatTest/unit_tests/expected_output/variants.re

```diff
@@ -269,9 +269,9 @@ let thisWontCompileButLetsSeeHowItFormats =
  * GADTs.
  */
 type term(_) =
-  | Int(int) : term(int)
+  | Int(int): term(int)
   | Add: term(((int, int) => int))
-  | App(term((('b) => 'a)), term('b)) : term('a);
+  | App(term((('b) => 'a)), term('b)): term('a);
```

Comment:
Commit https://github.com/facebook/reason/pull/1299/commits/0640e0dc1cffbf803c3c87c077cfef787bb36c0d prepare the ground for that. Your proposal can be parsed now, but is not printed yet.